### PR TITLE
Create organization profile readme

### PR DESCRIPTION
**Pull request by jtbandes**
Add a public org-level README which will appear at https://github.com/foxglove

https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile

### CODE REVIEW

**Review by amacneil — APPROVED** [review]

**Comment by defunctzombie** [comment]
This PR merged but I don't see any change to the organization page. Where would I see this?

**Comment by amacneil** [comment]
@defunctzombie view incognito or use this thing
image

**Comment by defunctzombie** [comment]
Thanks! had no idea there were different views. Looks great!

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[![Accelerate your robotics development](https://user-images.githubusercontent.com/14011012/195918769-5aaeedf3-5de2-48fb-951e-7399f2b9e190.png)](https://foxglove.dev)
+
+[<img align="right" width="400" src="https://github.com/foxglove/studio/raw/main/resources/screenshot.png">](https://github.com/foxglove/studio)
+
+Foxglove was founded to improve the development experience for robotics teams by bringing high quality tools to the robotics industry. Our mission is to reduce the time to market for our users, and accelerate the robotics revolution.
```
Comment (esthersweon) [inline profile/README.md:5]: "high-quality"

We use "robotics" 3 times in 2 sentences - could pare down.

Suggestion: "Foxglove provides high-quality tools that streamline the robotics development experience. By reducing the time to market for our users, we hope to accelerate industry progress towards a robotics revolution."

Comment (jtbandes) [reply profile/README.md:5]: I took this content directly from https://foxglove.dev/about.

Comment (esthersweon) [reply profile/README.md:5]: Oof got it - will revisit that page.

**Pull request merged**